tox.ini

```diff
@@ -27,13 +27,20 @@ envlist =
     #coverage-report
 
 [testenv]
-commands = coverage run -p -m pytest --durations=20 {posargs:tests} -m "not slow"
+commands =
+    # custom installation for PyG
```

Comment:
I hate this so much. Already discussion here https://github.com/rusty1s/pytorch_scatter/pull/268 on how to make some of these requirements easier to install...

I guess this is acceptable though, but optimally this would all be part of the actual dependencies for the package

Reply:
I do not like that either, but currently the only alternative is to not run the automated tests for PyG components.